.github/workflows/build_deps.yml

```diff
@@ -229,7 +229,7 @@ jobs:
             brew install gcc || true
             CC="/usr/local/bin/gcc"
             LD="/usr/local/bin/gcc"
-            CXX=="/usr/local/bin/g++"
+            CXX="/usr/local/bin/g++"
```

Comment:
This CXX change should be a separate commit and/or a separate PR.  Otherwise, blame will point to this commit, and reverting this commit will revert both changes.

Reply:
done